docs/ldml/tr35-dates.md

```diff
@@ -38,52 +38,53 @@ The LDML specification is divided into the following parts:
 
 ## <a name="Contents" href="#Contents">Contents of Part 4, Dates</a>
 
-*   1 [Overview: Dates Element, Supplemental Date and Calendar Information](#Overview_Dates_Element_Supplemental)
-*   2 [Calendar Elements](#Calendar_Elements)
-    *   2.1 [Elements months, days, quarters, eras](#months_days_quarters_eras)
-    *   2.2 [Elements monthPatterns, cyclicNameSets](#monthPatterns_cyclicNameSets)
-    *   2.3 [Element dayPeriods](#dayPeriods)
-    *   2.4 [Element dateFormats](#dateFormats)
-    *   2.5 [Element timeFormats](#timeFormats)
-    *   2.6 [Element dateTimeFormats](#dateTimeFormats)
-        *   2.6.1 [Element dateTimeFormat](#dateTimeFormat)
-            *   Table: [Date-Time Combination Examples](#Date_Time_Combination_Examples)
-        *   2.6.2 [Elements availableFormats, appendItems](#availableFormats_appendItems)
-            *   2.6.2.1 [Matching Skeletons](#Matching_Skeletons)
-            *   2.6.2.2 [Missing Skeleton Fields](#Missing_Skeleton_Fields)
-        *   2.6.3 [Element intervalFormats](#intervalFormats)
-*   3 [Calendar Fields](#Calendar_Fields)
-*   4 [Supplemental Calendar Data](#Supplemental_Calendar_Data)
-    *   4.1 [Calendar Data](#Calendar_Data)
-    *   4.2 [Calendar Preference Data](#Calendar_Preference_Data)
-    *   4.3 [Week Data](#Week_Data)
-        *   Table: [Week Designation Types](#Week_Designation_Types)
-    *   4.4 [Time Data](#Time_Data)
-    *   4.5 [Day Period Rule Sets](#Day_Period_Rule_Sets)
-        *   4.5.1 [Day Period Rules](#Day_Period_Rules)
-            *   4.5.1.1 [Fixed periods](#Fixed_periods)
```

Comment:
The formatting is inconsistent. In many cases it includes the number in the following anchor text, and in many cases it does not.

**[Turn on Hide Whitespace changes to do the review and this becomes far more obvious.]**

Reply:
Note: other than that the formatting looks pretty good. It would be nice to have the Table: back if that can be done.

Reply:
this probably points to inconsistencies in the source text, since this is an automated generation.

Table: will come back soon, but probably not with Table

Reply:
Ok, that makes sense. We should fix the inconsistency in another ticket. (Having them all adjacent in the TOC makes them obvious!)

Reply:
I suppose that we could do the same with "Table: ", that is, fix on the source line.

I presume in the production version the table captions will be in a larger font size.

Reply:
if we want to make that part of the title, we could. per the other ticket, the 'captions' are now just H6 (`######`).